src/bvals/comms/bnd_info.hpp

```diff
@@ -59,6 +59,7 @@ struct BndInfo {
   bool allocated = true;
   bool buf_allocated = true;
   int alloc_status;
+  bool same_to_same = false;
```

Comment:
This flag denotes that this buffer represents a communication from this block to itself and that the buffer does not need to be filled (a message just needs to be sent).